crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -17,13 +17,18 @@ pub trait Command: Send + Sync + 'static {
     fn write(self, world: &mut World);
 }
 
-/// A list of commands that modify a [`World`], running at the end of the stage where they
-/// have been invoked.
+/// A list of commands that runs at the end of the stage of the system that called them.
+///
+/// Each command can be used to modify the [`World`] for things like:
+/// * spawning entities
+/// * inserting components on new or existing entities
+/// * inserting resources
+/// * etc.
 ///
 /// # Usage
 ///
-/// `Commands` is a [`SystemParam`](crate::system::SystemParam), therefore it is declared
-/// as a function parameter:
+/// To use it, you can declare it as a function parameter of any system since it implements [`SystemParam`](crate::system::SystemParam).
+/// It's also important to declare `commands` as `mut`.
```

Comment:
```suggestion
/// Simply add `mut commands: Commands` as function argument to your system to get a copy of this struct that will be applied at the end of the current stage.
```

Reply:
I prefer avoiding using words like "simply" because it implies something is simple, but for some people it might not be as simple as it seems to us, but I like the overall direction of that sentence otherwise. For example, I've seen a few people be confused by `mut commands: Commands` vs `commands: &mut Commands`.

Reply:
Agreed.